packaging/installer/netdata-updater.sh

```diff
@@ -847,7 +847,7 @@ export NETDATA_LIB_DIR="${NETDATA_LIB_DIR:-${NETDATA_PREFIX}/var/lib/netdata}"
 # Grab the nightlies baseurl (defaulting to our Google Storage bucket)
 export NETDATA_NIGHTLIES_BASEURL="${NETDATA_NIGHTLIES_BASEURL:-https://storage.googleapis.com/netdata-nightlies}"
 
-if [ "${INSTALL_UID}" != "$(id -u)" ]; then
+if echo "$INSTALL_TYPE" | grep -qv ^binpkg && [ "${INSTALL_UID}" != "$(id -u)" ]; then
   fatal "You are running this script as user with uid $(id -u). We recommend to run this script as root (user with uid 0)"
```

Comment:
I think this check is _disconnected_ from the fatal message, perhaps we need just to check [ $(id -u) != "0" ] here.